rules/windows/builtin/win_susp_net_recon_activity.yml

```diff
@@ -22,13 +22,13 @@ detection:
     selection:
         - EventID: 4661
           ObjectType: 'SAM_USER'
-          ObjectName: 'S-1-5-21-*-500'
+          ObjectName|startswith: 'S-1-5-21-'
           AccessMask: '0x2d'
-        - EventID: 4661
-          ObjectType: 'SAM_GROUP'
-          ObjectName: 'S-1-5-21-*-512'
-          AccessMask: '0x2d'
-    condition: selection
+    selection2:
+          ObjectName|endswith: 
+            - '-500'
+            - '-512'
+    condition: selection and selection2
```

Comment:
the `ObjectType: 'SAM_GROUP'` is missed now. I would rather reconstruct it this way:

```
    selection:
        - EventID: 4661
          ObjectType:
            - 'SAM_USER'
            - 'SAM_GROUP'
          ObjectName|startswith: 'S-1-5-21-'
          ObjectName|endswith:
            - '-500'
            - '-512'
          AccessMask: '0x2d'
```

Reply:
Fixed ;)